_posts/2023/2023-08-07-play-in-a-sandbox-in-production.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Play in a sandbox in production"
+description: "Prevent disasters with a little caution"
+layout: article
+category: ruby
+image:
+  base: "2023/play-in-a-sandbox-in-production"
+  alt: "Toy truck in the sand"
+  credit: "Markus Spiske"
+  source: "https://unsplash.com/photos/KU3lOAiP-tQ"
+
+---
+
+As a developer you often have privileged access to the production environment of your application. This is often required to do bug fixes or to run commands for folks in your organisation.
+
+However with great power comes a great ability to shoot yourself in the foot. It’s all to easy to make damaging changes to user data without meaning it or 
```

Comment:
*too

Reply:
*or..?? Or?? The suspense is killing me. 😌

Reply:
```suggestion
However with great power comes a great ability to shoot yourself in the foot. It’s all to easy to make damaging changes to user data without meaning too.
```